content/developer/reference/javascript/generic_components.rst

```diff
@@ -44,4 +60,60 @@ Props
       - `boolean`
       - if true, the checkbox is disabled, otherwise it is enabled
 
+.. _generic_components/pager:
+
+Pager
+=====
+
+Location
+--------
+
+`@web/core/pager/pager`
+
+Description
+-----------
+
+The Pager is a small component to handle pagination.
+A page is defined by an `offset` and a `limit` (the size of the page).
+It displays the current page and the `total` number of elements, for instance, "9-12 / 20".
+In the previous example, `offset` is 8, `limit` is 4 and `total` is 20.
+It has two buttons ("Previous" and "Next") to navigate between pages.
```

Comment:
It's confusing to break the line in RST while the rendered paragraph in HTML will not contain anyline break. It's best practice to avoid breaking the line in RST if no line breaks are intended in HTML.